app/helpers/sessions_helper.rb

```diff
@@ -4,15 +4,36 @@ def log_in user
   end
 
   def current_user
-    @current_user ||= User.find_by id: session[:user_id]
+    if (user_id = session[:user_id])
```

Comment:
bỏ ngoặc được ko em

Reply:
Theo em là bỏ được chị. Nhưng do rubocop nó warning vì sợ mình nhầm với toán tử so sánh == 
Cậu lệnh if user_id = session[:user_id]
      @current_user ||= User.find_by id: user_id
Tương đương với.
user_id = session[:user_id]
if user_id
  @current_user ||= User.find_by id: user_id
end

Nếu không đúng thì chị reply lại giúp em để em note ạ. Thank chị :+1: